<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>The iSEEhex package • iSEEhex</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="The iSEEhex package">
<meta property="og:description" content="iSEEhex">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">iSEEhex</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/iSEEhex.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/iSEE/iSEEhex/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>The iSEEhex package</h1>
                        <h4 data-toc-skip class="author">Kevin
Rue-Albrecht</h4>
            <address class="author_afil">
      MRC Weatherall Institute of Molecular Medicine, University of
Oxford, Headington, Oxford OX3 9DS,
UK.<br><a class="author_email" href="mailto:#"></a><a href="mailto:kevin.rue-albrecht@imm.ox.ac.uk" class="email">kevin.rue-albrecht@imm.ox.ac.uk</a>
      </address>
                              <h4 data-toc-skip class="author">Federico
Marini</h4>
            <address class="author_afil">
      Institute of Medical Biostatistics, Epidemiology and Informatics
(IMBEI), MainzCenter for Thrombosis and Hemostasis (CTH),
Mainz<br><a class="author_email" href="mailto:#"></a><a href="mailto:marinif@uni-mainz.de" class="email">marinif@uni-mainz.de</a>
      </address>
                              <h4 data-toc-skip class="author">Charlotte
Soneson</h4>
            <address class="author_afil">
      Friedrich Miescher Institute for Biomedical Research, BaselSIB
Swiss Institute of
Bioinformatics<br><a class="author_email" href="mailto:#"></a><a href="mailto:charlottesoneson@gmail.com" class="email">charlottesoneson@gmail.com</a>
      </address>
                              <h4 data-toc-skip class="author">Aaron
Lun</h4>
                  <a class="author_email" href="mailto:#"></a><a href="mailto:infinite.monkeys.with.keyboards@gmail.com" class="email">infinite.monkeys.with.keyboards@gmail.com</a>
      
                  
            <h4 data-toc-skip class="date">1 June 2022</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/iSEE/iSEEhex/blob/HEAD/vignettes/iSEEhex.Rmd" class="external-link"><code>vignettes/iSEEhex.Rmd</code></a></small>
      <div class="hidden name"><code>iSEEhex.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p>The <em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em> package
<span class="citation">(Rue-Albrecht et al. 2018)</span> provides a
general and flexible framework for interactively exploring
<code>SummarizedExperiment</code> objects. However, in many cases, more
specialized panels are required for effective visualization of specific
data types. The <em><a href="https://bioconductor.org/packages/3.14/iSEEhex" class="external-link">iSEEhex</a></em>
package implements panels summarising data points in hexagonal bins,
that work directly in the <code>iSEE</code> application and can smoothly
interact with other panels.</p>
<p>We first load in the package:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/iSEE/iSEEhex" class="external-link">iSEEhex</a></span><span class="op">)</span></code></pre></div>
<p>All the panels described in this document can be deployed by simply
passing them into the <code><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE()</a></code> function via the
<code>initial=</code> argument, as shown in the following examples.</p>
<div class="section level3">
<h3 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h3>
<p>Let us prepare an example <em><a href="https://bioconductor.org/packages/3.14/SingleCellExperiment" class="external-link">SingleCellExperiment</a></em>
object.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">scRNAseq</span><span class="op">)</span>

<span class="co"># Example data ----</span>
<span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scRNAseq/man/ReprocessedData.html" class="external-link">ReprocessedAllenData</a></span><span class="op">(</span>assays<span class="op">=</span><span class="st">"tophat_counts"</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/base/class.html" class="external-link">class</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## [1] "SingleCellExperiment"</span>
<span class="co">## attr(,"package")</span>
<span class="co">## [1] "SingleCellExperiment"</span></code></pre>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://bioconductor.org/packages/scater/" class="external-link">scater</a></span><span class="op">)</span>
<span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scuttle/man/logNormCounts.html" class="external-link">logNormCounts</a></span><span class="op">(</span><span class="va">sce</span>, exprs_values<span class="op">=</span><span class="st">"tophat_counts"</span><span class="op">)</span>

<span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scater/man/runPCA.html" class="external-link">runPCA</a></span><span class="op">(</span><span class="va">sce</span>, ncomponents<span class="op">=</span><span class="fl">4</span><span class="op">)</span>
<span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scater/man/runTSNE.html" class="external-link">runTSNE</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span><span class="op">$</span><span class="va">ave_count</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowMeans</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">sce</span>, <span class="st">"tophat_counts"</span><span class="op">)</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span><span class="op">$</span><span class="va">n_cells</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowSums</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">sce</span>, <span class="st">"tophat_counts"</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">0</span><span class="op">)</span>
<span class="va">sce</span></code></pre></div>
<pre><code><span class="co">## class: SingleCellExperiment </span>
<span class="co">## dim: 20816 379 </span>
<span class="co">## metadata(2): SuppInfo which_qc</span>
<span class="co">## assays(2): tophat_counts logcounts</span>
<span class="co">## rownames(20816): 0610007P14Rik 0610009B22Rik ... Zzef1 Zzz3</span>
<span class="co">## rowData names(2): ave_count n_cells</span>
<span class="co">## colnames(379): SRR2140028 SRR2140022 ... SRR2139341 SRR2139336</span>
<span class="co">## colData names(23): NREADS NALIGNED ... passes_qc_checks_s sizeFactor</span>
<span class="co">## reducedDimNames(2): PCA TSNE</span>
<span class="co">## mainExpName: endogenous</span>
<span class="co">## altExpNames(1): ERCC</span></code></pre>
<p>Then, we create an <em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em> app
that compares the <code>ReducedDimensionHexPlot</code> panel – defined
in this package – to the standard <code>ReducedDimensionPlot</code>
defined in the <em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em>
package.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">initialPanels</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>
    <span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/ReducedDimensionPlot-class.html" class="external-link">ReducedDimensionPlot</a></span><span class="op">(</span>
        ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, ColorByFeatureName <span class="op">=</span> <span class="st">"Cux2"</span>, PanelWidth <span class="op">=</span> <span class="fl">6L</span><span class="op">)</span>,
    <span class="fu"><a href="../reference/ReducedDimensionHexPlot-class.html">ReducedDimensionHexPlot</a></span><span class="op">(</span>
        ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, ColorByFeatureName <span class="op">=</span> <span class="st">"Cux2"</span>, PanelWidth <span class="op">=</span> <span class="fl">6L</span>,
        BinResolution <span class="op">=</span> <span class="fl">30</span><span class="op">)</span>
<span class="op">)</span>
<span class="va">app</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE</a></span><span class="op">(</span>se <span class="op">=</span> <span class="va">sce</span>, initial <span class="op">=</span> <span class="va">initialPanels</span><span class="op">)</span></code></pre></div>
<p><img src="screenshots/reduced-dimension-hex-plot.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="further-reading">Further reading<a class="anchor" aria-label="anchor" href="#further-reading"></a>
</h3>
<div class="section level5">
<h5 class="unnumbered" id="where-can-i-find-a-comprehensive-introduction-to-isee">Where can I find a comprehensive introduction to
<em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em>?<a class="anchor" aria-label="anchor" href="#where-can-i-find-a-comprehensive-introduction-to-isee"></a>
</h5>
<p>The <em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em> package
contains several vignettes detailing the main functionality. You can
also take a look at this <a href="https://iSEE.github.io/iSEEWorkshop2019/index.html" class="external-link">workshop</a>.
A compiled version from the Bioc2019 conference (based on Bioconductor
release 3.10) is available <a href="http://biocworkshops2019.bioconductor.org.s3-website-us-east-1.amazonaws.com/page/iSEEWorkshop2019__iSEE-lab/" class="external-link">here</a>.</p>
</div>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></code></pre></div>
<pre><code><span class="co">## R version 4.1.3 (2022-03-10)</span>
<span class="co">## Platform: x86_64-pc-linux-gnu (64-bit)</span>
<span class="co">## Running under: Ubuntu 20.04.4 LTS</span>
<span class="co">## </span>
<span class="co">## Matrix products: default</span>
<span class="co">## BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so</span>
<span class="co">## </span>
<span class="co">## locale:</span>
<span class="co">##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span>
<span class="co">##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span>
<span class="co">##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span>
<span class="co">##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span>
<span class="co">##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span>
<span class="co">## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span>
<span class="co">## </span>
<span class="co">## attached base packages:</span>
<span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span>
<span class="co">## [8] base     </span>
<span class="co">## </span>
<span class="co">## other attached packages:</span>
<span class="co">##  [1] scater_1.22.0               ggplot2_3.3.6              </span>
<span class="co">##  [3] scuttle_1.4.0               scRNAseq_2.8.0             </span>
<span class="co">##  [5] iSEEhex_0.99.0              iSEE_2.6.0                 </span>
<span class="co">##  [7] SingleCellExperiment_1.16.0 SummarizedExperiment_1.24.0</span>
<span class="co">##  [9] Biobase_2.54.0              GenomicRanges_1.46.1       </span>
<span class="co">## [11] GenomeInfoDb_1.30.1         IRanges_2.28.0             </span>
<span class="co">## [13] S4Vectors_0.32.4            BiocGenerics_0.40.0        </span>
<span class="co">## [15] MatrixGenerics_1.6.0        matrixStats_0.62.0         </span>
<span class="co">## [17] BiocStyle_2.22.0           </span>
<span class="co">## </span>
<span class="co">## loaded via a namespace (and not attached):</span>
<span class="co">##   [1] utf8_1.2.2                    shinydashboard_0.7.2         </span>
<span class="co">##   [3] tidyselect_1.1.2              RSQLite_2.2.14               </span>
<span class="co">##   [5] AnnotationDbi_1.56.2          htmlwidgets_1.5.4            </span>
<span class="co">##   [7] grid_4.1.3                    BiocParallel_1.28.3          </span>
<span class="co">##   [9] Rtsne_0.16                    munsell_0.5.0                </span>
<span class="co">##  [11] ScaledMatrix_1.2.0            codetools_0.2-18             </span>
<span class="co">##  [13] ragg_1.2.2                    DT_0.23                      </span>
<span class="co">##  [15] miniUI_0.1.1.1                withr_2.5.0                  </span>
<span class="co">##  [17] colorspace_2.0-3              filelock_1.0.2               </span>
<span class="co">##  [19] highr_0.9                     knitr_1.39                   </span>
<span class="co">##  [21] shinyWidgets_0.7.0            GenomeInfoDbData_1.2.7       </span>
<span class="co">##  [23] bit64_4.0.5                   rprojroot_2.0.3              </span>
<span class="co">##  [25] vctrs_0.4.1                   generics_0.1.2               </span>
<span class="co">##  [27] xfun_0.31                     BiocFileCache_2.2.1          </span>
<span class="co">##  [29] R6_2.5.1                      doParallel_1.0.17            </span>
<span class="co">##  [31] ggbeeswarm_0.6.0              clue_0.3-61                  </span>
<span class="co">##  [33] rsvd_1.0.5                    AnnotationFilter_1.18.0      </span>
<span class="co">##  [35] bitops_1.0-7                  cachem_1.0.6                 </span>
<span class="co">##  [37] shinyAce_0.4.2                DelayedArray_0.20.0          </span>
<span class="co">##  [39] assertthat_0.2.1              promises_1.2.0.1             </span>
<span class="co">##  [41] BiocIO_1.4.0                  scales_1.2.0                 </span>
<span class="co">##  [43] beeswarm_0.4.0                gtable_0.3.0                 </span>
<span class="co">##  [45] beachmat_2.10.0               ensembldb_2.18.4             </span>
<span class="co">##  [47] rlang_1.0.2                   systemfonts_1.0.4            </span>
<span class="co">##  [49] GlobalOptions_0.1.2           splines_4.1.3                </span>
<span class="co">##  [51] rtracklayer_1.54.0            lazyeval_0.2.2               </span>
<span class="co">##  [53] hexbin_1.28.2                 BiocManager_1.30.18          </span>
<span class="co">##  [55] yaml_2.3.5                    GenomicFeatures_1.46.5       </span>
<span class="co">##  [57] httpuv_1.6.5                  tools_4.1.3                  </span>
<span class="co">##  [59] bookdown_0.26                 ellipsis_0.3.2               </span>
<span class="co">##  [61] jquerylib_0.1.4               RColorBrewer_1.1-3           </span>
<span class="co">##  [63] Rcpp_1.0.8.3                  sparseMatrixStats_1.6.0      </span>
<span class="co">##  [65] progress_1.2.2                zlibbioc_1.40.0              </span>
<span class="co">##  [67] purrr_0.3.4                   RCurl_1.98-1.6               </span>
<span class="co">##  [69] prettyunits_1.1.1             GetoptLong_1.0.5             </span>
<span class="co">##  [71] viridis_0.6.2                 fontawesome_0.2.2            </span>
<span class="co">##  [73] ggrepel_0.9.1                 cluster_2.1.3                </span>
<span class="co">##  [75] fs_1.5.2                      magrittr_2.0.3               </span>
<span class="co">##  [77] circlize_0.4.15               colourpicker_1.1.1           </span>
<span class="co">##  [79] ProtGenerics_1.26.0           hms_1.1.1                    </span>
<span class="co">##  [81] shinyjs_2.1.0                 mime_0.12                    </span>
<span class="co">##  [83] evaluate_0.15                 xtable_1.8-4                 </span>
<span class="co">##  [85] XML_3.99-0.9                  gridExtra_2.3                </span>
<span class="co">##  [87] shape_1.4.6                   compiler_4.1.3               </span>
<span class="co">##  [89] biomaRt_2.50.3                tibble_3.1.7                 </span>
<span class="co">##  [91] crayon_1.5.1                  htmltools_0.5.2              </span>
<span class="co">##  [93] mgcv_1.8-40                   later_1.3.0                  </span>
<span class="co">##  [95] DBI_1.1.2                     ExperimentHub_2.2.1          </span>
<span class="co">##  [97] dbplyr_2.1.1                  ComplexHeatmap_2.10.0        </span>
<span class="co">##  [99] rappdirs_0.3.3                Matrix_1.4-1                 </span>
<span class="co">## [101] cli_3.3.0                     parallel_4.1.3               </span>
<span class="co">## [103] igraph_1.3.1                  pkgconfig_2.0.3              </span>
<span class="co">## [105] pkgdown_2.0.3                 GenomicAlignments_1.30.0     </span>
<span class="co">## [107] xml2_1.3.3                    foreach_1.5.2                </span>
<span class="co">## [109] vipor_0.4.5                   bslib_0.3.1                  </span>
<span class="co">## [111] XVector_0.34.0                stringr_1.4.0                </span>
<span class="co">## [113] digest_0.6.29                 Biostrings_2.62.0            </span>
<span class="co">## [115] rmarkdown_2.14                rintrojs_0.3.0               </span>
<span class="co">## [117] DelayedMatrixStats_1.16.0     restfulr_0.0.13              </span>
<span class="co">## [119] curl_4.3.2                    shiny_1.7.1                  </span>
<span class="co">## [121] Rsamtools_2.10.0              rjson_0.2.21                 </span>
<span class="co">## [123] lifecycle_1.0.1               nlme_3.1-157                 </span>
<span class="co">## [125] jsonlite_1.8.0                BiocNeighbors_1.12.0         </span>
<span class="co">## [127] desc_1.4.1                    viridisLite_0.4.0            </span>
<span class="co">## [129] fansi_1.0.3                   pillar_1.7.0                 </span>
<span class="co">## [131] lattice_0.20-45               KEGGREST_1.34.0              </span>
<span class="co">## [133] fastmap_1.1.0                 httr_1.4.3                   </span>
<span class="co">## [135] interactiveDisplayBase_1.32.0 glue_1.6.2                   </span>
<span class="co">## [137] png_0.1-7                     iterators_1.0.14             </span>
<span class="co">## [139] BiocVersion_3.14.0            bit_4.0.4                    </span>
<span class="co">## [141] stringi_1.7.6                 sass_0.4.1                   </span>
<span class="co">## [143] blob_1.2.3                    textshaping_0.3.6            </span>
<span class="co">## [145] BiocSingular_1.10.0           AnnotationHub_3.2.2          </span>
<span class="co">## [147] memoise_2.0.1                 dplyr_1.0.9                  </span>
<span class="co">## [149] irlba_2.3.5</span></code></pre>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-iSEE-2018" class="csl-entry">
Rue-Albrecht, Kevin, Federico Marini, Charlotte Soneson, and Aaron T. L.
Lun. 2018. <span>“iSEE: Interactive SummarizedExperiment
Explorer.”</span> <em>F1000Research</em> 7 (June): 741. <a href="https://doi.org/10.12688/f1000research.14966.1" class="external-link">https://doi.org/10.12688/f1000research.14966.1</a>.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Kevin Rue-Albrecht, Charlotte Soneson, Federico Marini, Aaron Lun.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.3.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
