<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>The iSEEhex package • iSEEhex</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.4.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet">
<meta property="og:title" content="The iSEEhex package">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">iSEEhex</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.7.1</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item"><a class="nav-link" href="../articles/iSEEhex.html">Get started</a></li>
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="nav-item"><a class="nav-link" href="../news/index.html">Changelog</a></li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
<li class="nav-item"><a class="external-link nav-link" href="https://github.com/iSEE/iSEEhex/" aria-label="GitHub"><span class="fa fab fa-github fa-lg"></span></a></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>The iSEEhex package</h1>
                        <h4 data-toc-skip class="author">Kevin
Rue-Albrecht</h4>
            <address class="author_afil">
      MRC Weatherall Institute of Molecular Medicine, University of
Oxford, Headington, Oxford OX3 9DS,
UK.<br><a class="author_email" href="mailto:#"></a><a href="mailto:kevin.rue-albrecht@imm.ox.ac.uk" class="email">kevin.rue-albrecht@imm.ox.ac.uk</a>
      </address>
                              <h4 data-toc-skip class="author">Federico
Marini</h4>
            <address class="author_afil">
      Institute of Medical Biostatistics, Epidemiology and Informatics
(IMBEI), MainzCenter for Thrombosis and Hemostasis (CTH),
Mainz<br><a class="author_email" href="mailto:#"></a><a href="mailto:marinif@uni-mainz.de" class="email">marinif@uni-mainz.de</a>
      </address>
                              <h4 data-toc-skip class="author">Charlotte
Soneson</h4>
            <address class="author_afil">
      Friedrich Miescher Institute for Biomedical Research, BaselSIB
Swiss Institute of
Bioinformatics<br><a class="author_email" href="mailto:#"></a><a href="mailto:charlottesoneson@gmail.com" class="email">charlottesoneson@gmail.com</a>
      </address>
                              <h4 data-toc-skip class="author">Aaron
Lun</h4>
                  <a class="author_email" href="mailto:#"></a><a href="mailto:infinite.monkeys.with.keyboards@gmail.com" class="email">infinite.monkeys.with.keyboards@gmail.com</a>
      
                  
            <h4 data-toc-skip class="date">18 October 2024</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/iSEE/iSEEhex/blob/devel/vignettes/iSEEhex.Rmd" class="external-link"><code>vignettes/iSEEhex.Rmd</code></a></small>
      <div class="d-none name"><code>iSEEhex.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p>The <em><a href="https://bioconductor.org/packages/3.20/iSEE" class="external-link">iSEE</a></em> package
<span class="citation">(Rue-Albrecht et al. 2018)</span> provides a
general and flexible framework for interactively exploring
<code>SummarizedExperiment</code> objects. However, in many cases, more
specialized panels are required for effective visualization of specific
data types. The <em><a href="https://bioconductor.org/packages/3.20/iSEEhex" class="external-link">iSEEhex</a></em>
package implements panels summarising data points in hexagonal bins,
that work directly in the <code>iSEE</code> application and can smoothly
interact with other panels.</p>
<p>We first load in the package:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/iSEE/iSEEhex" class="external-link">iSEEhex</a></span><span class="op">)</span></span></code></pre></div>
<p>All the panels described in this document can be deployed by simply
passing them into the <code><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE()</a></code> function via the
<code>initial=</code> argument, as shown in the following examples.</p>
<div class="section level3">
<h3 id="example">Example<a class="anchor" aria-label="anchor" href="#example"></a>
</h3>
<p>Let us prepare an example <em><a href="https://bioconductor.org/packages/3.20/SingleCellExperiment" class="external-link">SingleCellExperiment</a></em>
object.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">scRNAseq</span><span class="op">)</span></span>
<span></span>
<span><span class="co"># Example data ----</span></span>
<span><span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scRNAseq/man/ReprocessedData.html" class="external-link">ReprocessedAllenData</a></span><span class="op">(</span>assays<span class="op">=</span><span class="st">"tophat_counts"</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/class.html" class="external-link">class</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "SingleCellExperiment"</span></span>
<span><span class="co">## attr(,"package")</span></span>
<span><span class="co">## [1] "SingleCellExperiment"</span></span></code></pre>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="http://bioconductor.org/packages/scater/" class="external-link">scater</a></span><span class="op">)</span></span>
<span><span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scuttle/man/logNormCounts.html" class="external-link">logNormCounts</a></span><span class="op">(</span><span class="va">sce</span>, exprs_values<span class="op">=</span><span class="st">"tophat_counts"</span><span class="op">)</span></span>
<span></span>
<span><span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/BiocSingular/man/runPCA.html" class="external-link">runPCA</a></span><span class="op">(</span><span class="va">sce</span>, ncomponents<span class="op">=</span><span class="fl">4</span><span class="op">)</span></span>
<span><span class="va">sce</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/scater/man/runTSNE.html" class="external-link">runTSNE</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span><span class="op">$</span><span class="va">ave_count</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowMeans</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">sce</span>, <span class="st">"tophat_counts"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">rowData</a></span><span class="op">(</span><span class="va">sce</span><span class="op">)</span><span class="op">$</span><span class="va">n_cells</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">rowSums</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">sce</span>, <span class="st">"tophat_counts"</span><span class="op">)</span> <span class="op">&gt;</span> <span class="fl">0</span><span class="op">)</span></span>
<span><span class="va">sce</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SingleCellExperiment </span></span>
<span><span class="co">## dim: 20816 379 </span></span>
<span><span class="co">## metadata(2): SuppInfo which_qc</span></span>
<span><span class="co">## assays(2): tophat_counts logcounts</span></span>
<span><span class="co">## rownames(20816): 0610007P14Rik 0610009B22Rik ... Zzef1 Zzz3</span></span>
<span><span class="co">## rowData names(2): ave_count n_cells</span></span>
<span><span class="co">## colnames(379): SRR2140028 SRR2140022 ... SRR2139341 SRR2139336</span></span>
<span><span class="co">## colData names(23): NREADS NALIGNED ... passes_qc_checks_s sizeFactor</span></span>
<span><span class="co">## reducedDimNames(2): PCA TSNE</span></span>
<span><span class="co">## mainExpName: endogenous</span></span>
<span><span class="co">## altExpNames(1): ERCC</span></span></code></pre>
<p>Then, we create an <em><a href="https://bioconductor.org/packages/3.20/iSEE" class="external-link">iSEE</a></em> app
that compares the <code>ReducedDimensionHexPlot</code> panel – defined
in this package – to the standard <code>ReducedDimensionPlot</code>
defined in the <em><a href="https://bioconductor.org/packages/3.20/iSEE" class="external-link">iSEE</a></em>
package.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">initialPanels</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/ReducedDimensionPlot-class.html" class="external-link">ReducedDimensionPlot</a></span><span class="op">(</span></span>
<span>        ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, ColorByFeatureName <span class="op">=</span> <span class="st">"Cux2"</span>, PanelWidth <span class="op">=</span> <span class="fl">6L</span><span class="op">)</span>,</span>
<span>    <span class="fu"><a href="../reference/ReducedDimensionHexPlot-class.html">ReducedDimensionHexPlot</a></span><span class="op">(</span></span>
<span>        ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, ColorByFeatureName <span class="op">=</span> <span class="st">"Cux2"</span>, PanelWidth <span class="op">=</span> <span class="fl">6L</span>,</span>
<span>        BinResolution <span class="op">=</span> <span class="fl">30</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="va">app</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE</a></span><span class="op">(</span>se <span class="op">=</span> <span class="va">sce</span>, initial <span class="op">=</span> <span class="va">initialPanels</span><span class="op">)</span></span></code></pre></div>
<p><img src="screenshots%2Freduced-dimension-hex-plot.png" width="100%"></p>
</div>
<div class="section level3">
<h3 id="further-reading">Further reading<a class="anchor" aria-label="anchor" href="#further-reading"></a>
</h3>
<div class="section level5">
<h5 class="unnumbered" id="where-can-i-find-a-comprehensive-introduction-to-isee">Where can I find a comprehensive introduction to
<em><a href="https://bioconductor.org/packages/3.20/iSEE" class="external-link">iSEE</a></em>?<a class="anchor" aria-label="anchor" href="#where-can-i-find-a-comprehensive-introduction-to-isee"></a>
</h5>
<p>The <em><a href="https://bioconductor.org/packages/3.20/iSEE" class="external-link">iSEE</a></em> package
contains several vignettes detailing the main functionality. You can
also take a look at this <a href="https://iSEE.github.io/iSEEWorkshop2019/index.html" class="external-link">workshop</a>.
A compiled version from the Bioc2019 conference (based on Bioconductor
release 3.10) is available <a href="http://biocworkshops2019.bioconductor.org.s3-website-us-east-1.amazonaws.com/page/iSEEWorkshop2019__iSEE-lab/" class="external-link">here</a>.</p>
</div>
</div>
</div>
<div class="section level2">
<h2 class="unnumbered" id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## R version 4.4.1 (2024-06-14)</span></span>
<span><span class="co">## Platform: x86_64-pc-linux-gnu</span></span>
<span><span class="co">## Running under: Ubuntu 22.04.5 LTS</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">## LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.20.so;  LAPACK version 3.10.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## time zone: UTC</span></span>
<span><span class="co">## tzcode source: system (glibc)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] scater_1.33.4               ggplot2_3.5.1              </span></span>
<span><span class="co">##  [3] scuttle_1.15.4              scRNAseq_2.19.1            </span></span>
<span><span class="co">##  [5] iSEEhex_1.7.1               iSEE_2.17.4                </span></span>
<span><span class="co">##  [7] SingleCellExperiment_1.27.2 SummarizedExperiment_1.35.4</span></span>
<span><span class="co">##  [9] Biobase_2.65.1              GenomicRanges_1.57.2       </span></span>
<span><span class="co">## [11] GenomeInfoDb_1.41.2         IRanges_2.39.2             </span></span>
<span><span class="co">## [13] S4Vectors_0.43.2            BiocGenerics_0.51.3        </span></span>
<span><span class="co">## [15] MatrixGenerics_1.17.0       matrixStats_1.4.1          </span></span>
<span><span class="co">## [17] BiocStyle_2.33.1           </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##   [1] splines_4.4.1            later_1.3.2              BiocIO_1.15.2           </span></span>
<span><span class="co">##   [4] bitops_1.0-9             filelock_1.0.3           tibble_3.2.1            </span></span>
<span><span class="co">##   [7] XML_3.99-0.17            lifecycle_1.0.4          httr2_1.0.5             </span></span>
<span><span class="co">##  [10] doParallel_1.0.17        lattice_0.22-6           ensembldb_2.29.1        </span></span>
<span><span class="co">##  [13] alabaster.base_1.5.9     magrittr_2.0.3           sass_0.4.9              </span></span>
<span><span class="co">##  [16] rmarkdown_2.28           jquerylib_0.1.4          yaml_2.3.10             </span></span>
<span><span class="co">##  [19] httpuv_1.6.15            DBI_1.2.3                RColorBrewer_1.1-3      </span></span>
<span><span class="co">##  [22] abind_1.4-8              zlibbioc_1.51.1          Rtsne_0.17              </span></span>
<span><span class="co">##  [25] AnnotationFilter_1.29.0  RCurl_1.98-1.16          rappdirs_0.3.3          </span></span>
<span><span class="co">##  [28] circlize_0.4.16          GenomeInfoDbData_1.2.13  ggrepel_0.9.6           </span></span>
<span><span class="co">##  [31] irlba_2.3.5.1            alabaster.sce_1.5.1      pkgdown_2.1.1           </span></span>
<span><span class="co">##  [34] codetools_0.2-20         DelayedArray_0.31.14     DT_0.33                 </span></span>
<span><span class="co">##  [37] tidyselect_1.2.1         shape_1.4.6.1            UCSC.utils_1.1.0        </span></span>
<span><span class="co">##  [40] viridis_0.6.5            ScaledMatrix_1.13.0      shinyWidgets_0.8.7      </span></span>
<span><span class="co">##  [43] BiocFileCache_2.13.2     GenomicAlignments_1.41.0 jsonlite_1.8.9          </span></span>
<span><span class="co">##  [46] GetoptLong_1.0.5         BiocNeighbors_1.99.2     iterators_1.0.14        </span></span>
<span><span class="co">##  [49] systemfonts_1.1.0        foreach_1.5.2            tools_4.4.1             </span></span>
<span><span class="co">##  [52] ragg_1.3.3               Rcpp_1.0.13              glue_1.8.0              </span></span>
<span><span class="co">##  [55] gridExtra_2.3            SparseArray_1.5.44       xfun_0.48               </span></span>
<span><span class="co">##  [58] mgcv_1.9-1               dplyr_1.1.4              HDF5Array_1.33.8        </span></span>
<span><span class="co">##  [61] gypsum_1.1.6             shinydashboard_0.7.2     withr_3.0.1             </span></span>
<span><span class="co">##  [64] BiocManager_1.30.25      fastmap_1.2.0            rhdf5filters_1.17.0     </span></span>
<span><span class="co">##  [67] fansi_1.0.6              shinyjs_2.1.0            rsvd_1.0.5              </span></span>
<span><span class="co">##  [70] digest_0.6.37            R6_2.5.1                 mime_0.12               </span></span>
<span><span class="co">##  [73] textshaping_0.4.0        colorspace_2.1-1         listviewer_4.0.0        </span></span>
<span><span class="co">##  [76] RSQLite_2.3.7            utf8_1.2.4               generics_0.1.3          </span></span>
<span><span class="co">##  [79] hexbin_1.28.4            rtracklayer_1.65.0       httr_1.4.7              </span></span>
<span><span class="co">##  [82] htmlwidgets_1.6.4        S4Arrays_1.5.11          pkgconfig_2.0.3         </span></span>
<span><span class="co">##  [85] gtable_0.3.5             blob_1.2.4               ComplexHeatmap_2.21.1   </span></span>
<span><span class="co">##  [88] XVector_0.45.0           htmltools_0.5.8.1        bookdown_0.41           </span></span>
<span><span class="co">##  [91] ProtGenerics_1.37.1      rintrojs_0.3.4           clue_0.3-65             </span></span>
<span><span class="co">##  [94] scales_1.3.0             alabaster.matrix_1.5.10  png_0.1-8               </span></span>
<span><span class="co">##  [97] knitr_1.48               rjson_0.2.23             nlme_3.1-166            </span></span>
<span><span class="co">## [100] curl_5.2.3               shinyAce_0.4.2           cachem_1.1.0            </span></span>
<span><span class="co">## [103] rhdf5_2.49.0             GlobalOptions_0.1.2      BiocVersion_3.20.0      </span></span>
<span><span class="co">## [106] parallel_4.4.1           miniUI_0.1.1.1           vipor_0.4.7             </span></span>
<span><span class="co">## [109] AnnotationDbi_1.67.0     restfulr_0.0.15          desc_1.4.3              </span></span>
<span><span class="co">## [112] pillar_1.9.0             grid_4.4.1               alabaster.schemas_1.5.0 </span></span>
<span><span class="co">## [115] vctrs_0.6.5              promises_1.3.0           BiocSingular_1.21.4     </span></span>
<span><span class="co">## [118] dbplyr_2.5.0             beachmat_2.21.6          xtable_1.8-4            </span></span>
<span><span class="co">## [121] cluster_2.1.6            beeswarm_0.4.0           evaluate_1.0.1          </span></span>
<span><span class="co">## [124] GenomicFeatures_1.57.1   cli_3.6.3                compiler_4.4.1          </span></span>
<span><span class="co">## [127] Rsamtools_2.21.2         rlang_1.1.4              crayon_1.5.3            </span></span>
<span><span class="co">## [130] ggbeeswarm_0.7.2         fs_1.6.4                 viridisLite_0.4.2       </span></span>
<span><span class="co">## [133] alabaster.se_1.5.3       BiocParallel_1.39.0      munsell_0.5.1           </span></span>
<span><span class="co">## [136] Biostrings_2.73.2        lazyeval_0.2.2           colourpicker_1.3.0      </span></span>
<span><span class="co">## [139] Matrix_1.7-0             ExperimentHub_2.13.1     bit64_4.5.2             </span></span>
<span><span class="co">## [142] Rhdf5lib_1.27.0          KEGGREST_1.45.1          shiny_1.9.1             </span></span>
<span><span class="co">## [145] highr_0.11               alabaster.ranges_1.5.2   AnnotationHub_3.13.3    </span></span>
<span><span class="co">## [148] fontawesome_0.5.2        igraph_2.0.3             memoise_2.0.1           </span></span>
<span><span class="co">## [151] bslib_0.8.0              bit_4.5.0</span></span></code></pre>
</div>
<div class="section level2">
<h2 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h2>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-iSEE-2018" class="csl-entry">
Rue-Albrecht, Kevin, Federico Marini, Charlotte Soneson, and Aaron T. L.
Lun. 2018. <span>“iSEE: Interactive SummarizedExperiment
Explorer.”</span> <em>F1000Research</em> 7 (June): 741. <a href="https://doi.org/10.12688/f1000research.14966.1" class="external-link">https://doi.org/10.12688/f1000research.14966.1</a>.
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Kevin Rue-Albrecht, Charlotte Soneson, Federico Marini, Aaron Lun.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

    </footer>
</div>





  </body>
</html>
